src/pages/Defi/components/OverviewHeader.tsx

```diff
@@ -61,7 +63,10 @@ export const OverviewHeader = ({
           gridGap={{ base: 0, lg: 6 }}
         >
           <EarnStat label='defi.walletBalance' value={walletBalanceWithoutEarn} />
-          <EarnStat label='defi.earnBalance' value={earnBalance.totalEarningBalance} />
+          <EarnStat
+            label='defi.earnBalance'
+            value={bnOrZero(earnBalance.totalEarningBalance).plus(bnOrZero(lpBalance)).toString()}
```

Comment:
- `earnBalance.totalEarningBalance` is already `bnOrZero().toString()`
- lpbalance has a default value of '0'
```suggestion
            value={bn(earnBalance.totalEarningBalance).plus(bn(lpBalance)).toString()}
```